examples/nidcpower_source_dc_voltage/teststand_fixture.py

```diff
@@ -6,6 +6,9 @@
 
 import ni_measurementlink_service as nims
 
+# To use a physical NI SMU instrument, set this to False.
+USE_SIMULATION = True
```

Comment:
It's annoying that this constant is duplicated between `measurement.py` and `teststand_fixture.py`.

`from measurement import USE_SIMULATION` might work in this PR, but it wouldn't work in the `multiple_measurements` example from https://github.com/ni/measurementlink-python/pull/256

Moving the constant to `_helpers.py` might work, but I don't want to encourage people to customize `_helpers.py`.

Moving the TestStand version to a sequence file global might make sense.